# Debug message for cone seeking
Header header

# Throttle and steering as percentage
uint8 throttle
int8 steering
uint8 confidence

# Cone distance
uint16 h
uint16 d

# Area of cone
int32 area
